hazelcast/src/main/java/com/hazelcast/spi/impl/BasicOperationScheduler.java

```diff
@@ -505,22 +508,26 @@ private void processPriorityMessages() {
             }
         }
 
-        @edu.umd.cs.findbugs.annotations.SuppressWarnings({"VO_VOLATILE_INCREMENT" })
+        @edu.umd.cs.findbugs.annotations.SuppressWarnings({"VO_VOLATILE_INCREMENT"})
         private void process(Object task) {
```

Comment:
I would set 'currentOperation' directly in the beginning of the method and slap a try/finally around it so it is unset in the finally clause.